src/styles/card.css

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: var(--card-width);
+  min-height: var(--card-height);
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  background: rgb(255, 255, 255);
+  border-radius: 5px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+}
+
+.card__imac--bottom {
+  margin-bottom: 40px;
+}
```

Comment:
add that to your element, and remove modifier, modifier should describe state, not a default behavior, and fix it for all these occurences